components/commons/modals/FolderAddInputModal.tsx

```diff
@@ -0,0 +1,19 @@
+import { InputModalBox, ModalLayout, GradientButton } from "./styledModal";
+import CloseButton from "./CloseButton";
+
+function FolderAddInputModal({ closeModal }: { closeModal: () => void }) {
```

Comment:
어떤 컴포넌트가 호출하도록 전달받는 prop은 on- 접두사를 붙여서 네이밍합니다.
```suggestion
function FolderAddInputModal({ onClose }: { onClose: () => void }) {
```